ttkwidgets/frames/vnotebook.py

```diff
@@ -0,0 +1,312 @@
+"""
+Author: RedFantom
+License: GNU GPLv3, as in LICENSE.md
+Copyright (C) 2018 RedFantom
+"""
+# Basic UI imports
+import tkinter as tk
+from tkinter import ttk
+from tkinter.font import Font
+
+
+class VNotebook(ttk.Frame):
+    """
+    Notebook with vertical tabs. Does not actually use the 
+    :class:`ttk.Notebook` widget, but a set of Toolbutton-styles 
+    Radiobuttons to select a Frame of a set. Provides an interface that 
+    behaves like a normal :class:`ttk.Notebook` widget.
+    """
+
+    options = [
+        # Notebook options
+        "cursor",
+        "padding",
+        "style",
+        "takefocus",
+        # VNotebook options
+        "compound",
+        "callback",
+    ]
+
+    tab_options = [
+        "compound",
+        "padding",
+        "sticky",
+        "image",
+        "text",
+        "underline",
+        "font",
+        "id",
+    ]
+
+    def __init__(self, master, **kwargs):
+        """
+        Create a VNotebook.
+        
+        :param cursor: Cursor set upon hovering Buttons
+        :type cursor: str
+        :param padding: Amount of pixels between the Buttons
+        :type padding: int
+        :param compound: Location of the Buttons
+        :type compound: str
+        :param callback: Callback called upon change of Frame
+        :type callback: callable
```

Comment:
By the way, wouldn't it be more consistent with the `ttk.Notebook` widget to use a virtual event, e.g. `<<VNotebookTabChanged>>` instead of a callback?

Reply:
I agree. A virtual event should be the way to go as it's more consistent with the rest of tkinter and ttk. It's also much easier to implement, and easier to document. Now, what data should the event argument hold in that case?

Reply:
When I make tkinter print the `<<NotebokkTabChanged>>` event, I get `<VirtualEvent event x=0 y=0>` so I think we don't have to pass any specific argument to the event, the new tab id and everything else can be recovered from the notebook's methods.

Reply:
I removed the 'callback' argument from the documentation, there didn't seem to be any code to remove with that change. I'll let you handle the event generation since you seem to be doing that.